README.md

```diff
@@ -1,5 +1,306 @@
+
+This portion of the application trio focuses on what is necessary to manipulate the state of these services on Azure directly. The collection you are viewing is to document all of the available calls one can make to this service once it is deployed online. It is intended to be published on Azure Functions, although, it could be changed slightly to work on other serverless platforms, such as AWS Lambda.
+
+Please note that this application is designed to create and destroy resources on Azure in a completely self-contained manner. It does not require an administrator to change or manage these resources in any way beyond what is required for the [installation](#Installation). Manually changing these resources in Azure after the application is running may cause interference, resulting in an application malfunction. If you want to manually manage your own Media Services resources, it is recommended that you create a separate Media Services instance and leave the one used by this application to run on its own.
+
+The impetus of this application was to create an easy way to spin up Media Services resources on Azure before a broadcast and spin them down again after the show ends. This helps minimize unnecessary billing on our Azure account and puts the power of service management into the hands of the broadcaster. To reduce the costs associated with streaming on Azure, note that ALL resources created for a live streaming session will be DESTROYED when the application stops them. That includes the [video buffer stored in the archive window](#LIVE_STREAMING_API_ARCHIVE_WINDOW_LENGTH). If you want to retain a copy of your live streaming session, it is recommended that the broadcaster records it locally before shutting down the stream and stopping the media resources.
+
+## Basic Definitions
+
+This project controls two major resource types on Azure Media Services: Streaming Endpoints and Live Events. Here are their definitions:
+
+- **Live Events:** A channel that a broadcaster will reuse to publish similar events over and over again.
+- **Streaming Endpoint:** The ingest mechanism on Azure Media Services which allows a broadcaster to connect their streaming software of choice, such as [Wirecast](https://www.telestream.net/wirecast/) or [OBS Studio](https://obsproject.com/), and begin sending a live stream.
+
+## Installation
+
+As mentioned before, this application is intended to run on Azure Functions. Therefore, the instructions are geared toward the process of installing the API there. Overall, there are several significant phases to the installation:
+
+1. [Setting up the Media Services resources on Azure](#Setting-up-the-Media-Services-Resources-on-Azure)
+1. [Creating the Azure Functions application](#Creating-the-Azure-Functions-Application)
+1. [Setting up the environment variables](#Setting-up-the-Environment-Variables)
+1. [Deploying the application](#Deploying-the-Application)
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `4`, was `1`](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5728997)